include/irr/asset/normal_quantization.h

```diff
@@ -160,7 +192,7 @@ namespace asset
         uint32_t bestFit = snormVec[0]|(snormVec[1]<<quantizationBits)|(snormVec[2]<<(quantizationBits*2u));
 
 		dummySearchVal.value = bestFit;
-        normalCacheFor2_10_10_10Quant.insert(found,dummySearchVal);
+        normalCacheFor2_10_10_10QuantUm.insert(std::make_pair(found->first, bestFit));
```

Comment:
why are you inserting into the cache what you've already found? Your key needs to be `projFitting` that you've tried searching with

For the `std::vector::insert` function the iterator is a placement hint.